addons/website_crm_partner_assign/controllers/main.py

```diff
@@ -227,6 +227,8 @@ def partners(self, country=None, grade=None, page=0, **post):
         base_partner_domain = [('is_company', '=', True), ('grade_id', '!=', False), ('website_published', '=', True)]
         if not request.env['res.users'].has_group('website.group_website_restricted_editor'):
             base_partner_domain += [('grade_id.website_published', '=', True)]
+        country_domain = list(base_partner_domain)
+        grade_domain = list(base_partner_domain)
```

Comment:
Testing the commit, I really don't understand why we want that (see other comment) but if we really do want that, it should be explained here in the code too.